tests/api-diff/type_tests.t

```diff
@@ -70,5 +70,6 @@ Run api-watcher on the two cmi files, there should be a difference
   diff module Modify_type:
   -type t = int
   +type t = float
+   
```

Comment:
Any idea why this new line is added (it seems there is a lot of new lines at the end of each diff)

Reply:
Yes. These are only there when both type_kind aren't changed and they are abstract, in that case there we print a newline for the unchanged type_kind for context.

Reply:
But is there any explanation why we would want a blank line there? I think it would make more sense not to print an empty line in this case!

Reply:
I totally agree. But here we have a Same hunk, which has an empty string underneath, to get rid of this newline we need to change the representation a bit, to have the Same hunk hold either a string option or a sting list. I am not sure if it's worth the trouble since we'll change the representation to have inline diffs very soon.

Reply:
Context lines will still be there even with inline change. Why not remove the `Same` hunk altogether here?